File: api-server/src/main/java/com/talentlink/talentlink/favorite/dto/FavoriteRequest.java
package com.talentlink.talentlink.favorite.dto;

import lombok.Getter;

import java.time.LocalDateTime;

@Getter
public class FavoriteRequest {
    private Long id;  // 판매글일 때만
    private String type;  // "buy" or "sell"
    private String userId;
    private String writerNickname;
    private Long buyId;   // 구매글일 때만
    private Long sellId;  // 판매글일 때만
}

File: front-server/app/src/main/java/com/example/talent_link/ui/Favorite/dto/FavoriteRequest.kt
package com.example.talent_link.ui.Favorite.dto

data class FavoriteRequest(
    val id: Long?,             // sell일 때만
    val type: String,          // "buy" or "sell"
    val userId: String,
    val writerNickname: String?,
    val buyId: Long?,          // buy일 때만
    val sellId: Long?          // sell일 때만
)